dataextractor.py

```diff
@@ -0,0 +1,300 @@
+#Importing
```

Comment:
Should start with a paragraph describing what the .py does. Possibly author information, group authorship --- or not. Date last updated.